src/components/table.js

```diff
@@ -250,11 +262,11 @@ export class GridTable extends Component {
   }
 
   scrollToTop() {
-    this.body.current.scrollToPosition({ scrollTop: 0 })
+    this.body.current.scrollToPosition({ scrollTop: 0, scrollLeft: 0 })
   }
 
   render() {
-    const { width, height, rowCount, columns, cellStyle } = this.props
+    const { width, height, rowCount, columns, cellStyle, onScroll: customOnScroll = _.identity } = this.props
```

Comment:
Could be `_.noop` instead.

Reply:
When I replaced _.identity with _.noop, this.body.current became undefined--should I be using _.noop differently?